<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>login_widget</class>
 <widget class="QWidget" name="login_widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>160</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>320</width>
    <height>160</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>320</width>
    <height>160</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Molo</string>
  </property>
  <widget class="QLabel" name="login_label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Log in</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="user_id_input">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>281</width>
     <height>28</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>User ID</string>
   </property>
  </widget>
  <widget class="QPushButton" name="login_btn">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>120</y>
     <width>84</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Log in</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="user_pw_input">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>281</width>
     <height>28</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
   <property name="placeholderText">
    <string>Password</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="remember_check">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>141</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Remember me?</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="tristate">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="error_msg">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>30</y>
     <width>171</width>
     <height>20</height>
    </rect>
   </property>
   <property name="palette">
    <palette>
     <active>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Text">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>252</red>
         <green>175</green>
         <blue>62</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="PlaceholderText">
       <brush brushstyle="SolidPattern">
        <color alpha="128">
         <red>252</red>
         <green>175</green>
         <blue>62</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Text">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>252</red>
         <green>175</green>
         <blue>62</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="PlaceholderText">
       <brush brushstyle="SolidPattern">
        <color alpha="128">
         <red>252</red>
         <green>175</green>
         <blue>62</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>190</red>
         <green>190</green>
         <blue>190</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Text">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>190</red>
         <green>190</green>
         <blue>190</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="PlaceholderText">
       <brush brushstyle="SolidPattern">
        <color alpha="128">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
     </disabled>
    </palette>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <zorder>login_label</zorder>
  <zorder>login_btn</zorder>
  <zorder>remember_check</zorder>
  <zorder>error_msg</zorder>
  <zorder>user_id_input</zorder>
  <zorder>user_pw_input</zorder>
 </widget>
 <tabstops>
  <tabstop>user_id_input</tabstop>
  <tabstop>user_pw_input</tabstop>
  <tabstop>remember_check</tabstop>
  <tabstop>login_btn</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
